dartagnan/src/test/java/com/dat3m/dartagnan/c/C11OrigLFDSTest.java

```diff
@@ -0,0 +1,90 @@
+package com.dat3m.dartagnan.c;
+
+import com.dat3m.dartagnan.configuration.Arch;
+import com.dat3m.dartagnan.configuration.OptionNames;
+import com.dat3m.dartagnan.utils.Result;
+import com.dat3m.dartagnan.utils.rules.Provider;
+import com.dat3m.dartagnan.utils.rules.Providers;
+import com.dat3m.dartagnan.verification.solving.AssumeSolver;
+import com.dat3m.dartagnan.verification.solving.RefinementSolver;
+import com.dat3m.dartagnan.wmm.Wmm;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.sosy_lab.common.configuration.Configuration;
+import org.sosy_lab.common.configuration.InvalidConfigurationException;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import static com.dat3m.dartagnan.configuration.Arch.C11;
+import static com.dat3m.dartagnan.utils.ResourceHelper.getTestResourcePath;
+import static com.dat3m.dartagnan.utils.Result.FAIL;
+import static com.dat3m.dartagnan.utils.Result.PASS;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(Parameterized.class)
+public class C11OrigLFDSTest extends AbstractCTest {
+
+    public C11OrigLFDSTest(String name, Arch target, Result expected) {
+        super(name, target, expected);
+    }
+
+    @Override
+    protected Provider<String> getProgramPathProvider() {
+        return Provider.fromSupplier(() -> getTestResourcePath("lfds/" + name + ".ll"));
+    }
+
+    @Override
+    protected long getTimeout() {
+        return 600000;
+    }
+
+    protected Provider<Integer> getBoundProvider() {
+        return Provider.fromSupplier(() -> 2);
+    }
+
+    @Override
+    protected Configuration getConfiguration() throws InvalidConfigurationException {
+        return Configuration.builder()
+                .copyFrom(super.getConfiguration())
+                .setOption(OptionNames.INIT_DYNAMIC_ALLOCATIONS, "true")
+                .build();
+    }
+
+    @Override
+    protected Provider<Wmm> getWmmProvider() {
+        return Providers.createWmmFromName(() -> "c11-orig");
+    }
+
+    @Parameterized.Parameters(name = "{index}: {0}, target={1}")
+    public static Iterable<Object[]> data() throws IOException {
+        // Commented ones take too long ATM
+        return Arrays.asList(new Object[][]{
+                // {"dglm", C11, UNKNOWN},
+                {"dglm-CAS-relaxed", C11, FAIL},
+                // {"ms", C11, UNKNOWN},
+                {"ms-CAS-relaxed", C11, FAIL},
+                // {"treiber", C11, UNKNOWN},
+                {"treiber-CAS-relaxed", C11, FAIL},
```

Comment:
Have you tried enabling these test? I think back in the day we commented them out because it took too long, but performance has improve since then

Reply:
I have not tried it. I think enabling CAAT for C11 was already a change that I should not have made in this PR. These changes are orthogonal to the PR and would better be done separately.

Reply:
I agree. I just noticed this (did not even remembered it) while reviewing the diff in the code